app/controllers/api/v1/orders_controller.rb

```diff
@@ -59,6 +59,26 @@ def show
           exclude_stockit_set_item: true
       end
 
+      def start_process
```

Comment:
Thanks @steveyken , we don't have methods for each transition. It's just a name of controller method.
We've directly called `@order.start_processing` which is a state event and not a method.

Reply:
Sorry, I wasn't very clear before, I was thinking instead of controller actions for every transition, why don't we just have one controller action and pass the transition as a parameter. E.g.

```
OrdersController
    def transition
      @transition = params['transition']   # e.g. 'start_processing'
      if @order.state_events.include?(@transition)
        @order.fire_state_event(@transition)
      end
      ...
    end
```

Reply:
Thanks @steveyken , looks much better.
Will do it tonight. :)